understanding/21/orientation.html

```diff
@@ -15,10 +15,10 @@ <h1>
 		</h1>
 		<section id="intent">
 			<h2>Intent of this Success Criterion</h2>
-			<p>The intent of this Success Criterion is to ensure that content displays in the orientation (portrait or landscape) preferred by the user. Some websites and applications automatically set the screen to a particular display orientation and expect
+			<p>The intent of this Success Criterion is to ensure that content displays in the orientation (portrait or landscape) preferred by the user. Some websites and applications automatically set and lock the screen to a particular display orientation and expect
```

Comment:
I know that we discussed whether this is "locking orientation" or "not enabling awareness of changes in orientation". I think that the paragraph is clear without the "and lock" and the "not locking the" (below), as it says that both need to be supported. But I don't care that much, so whatever people decide...

Reply:
From meeting: Happy with 'set and lock', would like to add this on line 19: 
"...their device to match, but this can create problems. Some users..."